blocks/filter.py

```diff
@@ -88,12 +96,20 @@ class VariableFilter(object):
 
     """
     def __init__(self, roles=None, bricks=None, each_role=False, name=None,
-                 application=None):
+                 name_re=None, applications=None, one=True):
         self.roles = roles
-        self.bricks = bricks
+        if not isinstance(bricks, list):
```

Comment:
Just to clarify: This is for duck typing reasons. Right now if someone passes e.g. a `tuple` the code breaks.